src/cdomain/value/cdomains/intDomain0.ml

```diff
@@ -319,7 +321,7 @@ struct
   let is_bot x = I.is_bot x.v
   let top_of ikind = { v = I.top_of ikind; ikind}
   let top () = failwith "top () is not implemented for IntDomLifter."
-  let is_top x = I.is_top x.v
+  let is_top _ = false
```

Comment:
Maybe this should raise now directly (as the inner implementations used to do)? It doesn't do what one would expect it to do, and any usage is bad.